### ISSUE TITLE
Version 15 no longer shows disconnected icons in the panel

### ISSUE THREAD

**Issue by kirbyfan64** [andyholmes/gnome-shell-extension-gsconnect#324]
I have my Display Mode set to panel. With previous versions, if my device wasn't connected, a grayed-out icon would appear in the panel, and I could click the icon within to try to reconnect. However, with version 15, there's...nothing. It was pretty helpful because my phone's Wi-Fi connection drops frequently, and I could easily just go there to try reconnecting.

**Comment by kirbyfan64** [comment]
Does the reconnect shortcut work if a connection hasn't been established yet on this session? It didn't work for me and I had to restart the GSConnect service.

**Comment by andyholmes** [comment]
Yep, it does the same thing that the button always did, which is to send a UDP packet to the last known IP address of the device.

Whether the device is still at that address, or will respond (phone is asleep), is up to the device. It's also the case that KDE Connect has a 30s keepalive for connections. It certainly possible GSConnect has recognized the connection is broken, but not the phone (or vice-versa).

**Comment by kirbyfan64** [comment]
Okay, I think I actually found the issue... I wasn't seeing the icon appear, so I figured it wasn't getting connected unless I restarted the service.

However, today when my reconnect shortcut didn't work, I just left it and moved on. A few hours later, I realized *I've been getting phone notifications anyway*. When I checked, sure enough, GSConnect was connected but the icon wasn't present until I restarted the service.
